department-of-veterans-affairs/va.gov-cms
# VAGOV-6102: Updating authored on perms for content editor role.

## What this does
Allows content editor to change the authored on date for Stories, Events, Publications, News releases, and VAMC banner alerts

## To test
 - Go to an existing story / event / publication / news release / banner alert
 - Visually verify you can see the authored on field, and change the time
 - Verify that you can save and update the node with the altered time.

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/72470861-906ba680-37af-11ea-802e-9daa7d743142.png)
![image](https://user-images.githubusercontent.com/2404547/72470876-95c8f100-37af-11ea-8f95-2a42dc80a06c.png)
